Camera2Extensions/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -43,6 +43,11 @@
             app:argType="string"
             app:nullable="false"/>
 
+        <argument
+            android:name="hdr"
```

Comment:
nit: hdr might be a bit confusing; maybe we rename this to jpeg_r ?

Reply:
Done